packages/@aws-cdk/aws-lambda/lib/code.ts

```diff
@@ -160,6 +162,54 @@ export class InlineCode extends Code {
   }
 }
 
+/**
+ * Bundle options
+ */
+export interface BundleOptions {
+  /**
+   * The Docker image where the command will run.
+   */
+  readonly image: DockerImage;
+
+  /**
+   * The command to run in the container.
+   */
+  readonly command: string[];
+
+  /**
+   * Docker volumes to mount.
+   *
+   * @default - The path to the asset file or directory is mounted at /asset
+   */
+  readonly volumes?: DockerVolume[];
+
+  /**
+   * The environment variables to pass to the container.
+   *
+   * @default - no environment variables.
+   */
+  readonly environment?: { [key: string]: string; };
+
+  /**
+   * Working directory inside the container.
+   *
+   * @default - the `containerPath` of the first mounted volume.
+   */
+  readonly workingDirectory?: string;
```

Comment:
I think this should just be `/asset` by default unless the user specifies something.